components/camel-restlet/pom.xml

```diff
@@ -161,5 +161,21 @@
     </dependency>
 
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
```

Comment:
It would be nice to use maven-jar-plugin-version from the parent/pom.xml

Reply:
Sorry, I removed it, because it was code which shouldn't be here.